File: MODUL1/PraktikumC/PRAK110-2310817110004-AbdurrahmanGilangHarjuna.c
#include <stdio.h>
#include <math.h>

int main() {
    float alas = 5;
    float tinggi = 12;
    float sisiA, sisiB, sisiC, keliling, luas;
    sisiA = alas;
    sisiB = tinggi;
    sisiC = hypot(alas, tinggi);
    keliling = sisiA + sisiB + sisiC;
    luas = 0.5 * alas * tinggi;
    printf("Diketahui:\n");
    printf("Alas = %.fcm\n", alas);
    printf("Tinggi = %.fcm\n", tinggi);
    printf("\nJawab:\n");
    printf("Sisi A = %.f cm\n", sisiA);
    printf("Sisi B = %.f cm\n", sisiB);
    printf("Sisi C = %.f cm\n", sisiC);
    printf("Keliling = %.f cm\n", keliling);
    printf("Luas = %.f cm\n", luas);
    return 0;
}

File: MODUL1/PraktikumPy/PRAK110-2310817110004-Abdurrahman Gilang Harjuna.py
import math
alas = 5
tinggi = 12
sisiA = alas
sisiB = tinggi
sisiC = math.hypot(alas, tinggi)
keliling = sisiA + sisiB + sisiC
luas = 0.5 * alas * tinggi
print("Diketahui:")
print(f"Alas = {alas}cm")
print(f"Tinggi= {tinggi}cm\n")
print("Jawab:")
print(f"Sisi A = {sisiA:.0f} cm")
print(f"Sisi B = {sisiB:.0f} cm")
print(f"Sisi C = {sisiC:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm")
